components/org.wso2.carbon.identity.api.server.idv.provider/org.wso2.carbon.identity.api.server.idv.provider.v1/src/main/resources/idv-provider.yaml

```diff
@@ -363,10 +363,10 @@ components:
         id:
           type: string
           example: 123e4567-e89b-12d3-a456-556642440000
-        Type:
+        type:
```

Comment:
Update the parameter name at https://github.com/wso2/identity-api-server/blob/4f708a27877f20028174a3e7db91b43dea063565/components/org.wso2.carbon.identity.api.server.idv.provider/org.wso2.carbon.identity.api.server.idv.provider.v1/src/main/resources/idv-provider.yaml#L410C8-L411C15

Reply:
Fixed with 9df161ac012f91dd7e36d8e568d2587183b1b6d4